amethyst_error/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "amethyst_error"
+version = "0.1.0"
+authors = ["John-John Tedro <udoprog@tedro.se>"]
+readme = "README.md"
+description = """
+Internal error handling for Amethyst.
+"""
+keywords = ["game", "error", "amethyst"]
+categories = ["error"]
+
+documentation = "https://www.amethyst.rs/doc/master/doc/amethyst/"
+homepage = "https://www.amethyst.rs/"
+repository = "https://github.com/amethyst/amethyst"
+
+[badges]
+appveyor = { repository = "amethyst/amethyst", branch = "master" }
+travis-ci = { repository = "amethyst/amethyst" }
+
+[dependencies]
+backtrace = {version = "0.3.9", optional = true}
+
+[features]
+default = ["backtrace"]
```

Comment:
Code won't capture backtraces without missing `std` feature.

Reply:
Backtrace is no longer optional.